scripts/deploy.ts

```diff
@@ -98,6 +100,7 @@ function makeCommonPre(argv: string[], buildObj: BuildObj) {
   buildObj.projectName = argv[2]
   buildObj.guiPlatformDir = buildObj.guiDir + '/' + buildObj.platformType
   buildObj.tmpDir = `${buildObj.guiDir}/temp`
+  buildObj.buildArchivesDir = '/Users/jenkins/buildArchives'
```

Comment:
Shouldn't we get the user dir from the environment. It'd be a rare case, so probably not worthwhile, but could the build server username change from `jenkins`?